.github/workflows/docker-publish.yml

```
@@ -0,0 +1,30 @@
+          username: ${{ secrets.AZURE_ACR_USERNAME }}
+          password: ${{ secrets.AZURE_ACR_PASSWORD }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          tags: |
+            crretoxmas2024.azurecr.io/ithaka-frontend:latest
+          cache-from: type=registry,ref=crretoxmas2024.azurecr.io/ithaka-frontend:cache
+          cache-to:   type=registry,ref=crretoxmas2024.azurecr.io/ithaka-frontend:cache,mode=max
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)